### PR TITLE
Refactor script args

### PR DESCRIPTION
**Pull request by eartser**

### CODE REVIEW

File: scripts/src/utils/models/script_parameters.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from argparse import Namespace
+from dataclasses import dataclass
+
+from src.utils.file_utils import get_absolute_path
+
+
+@dataclass
+class BaseRunnerParameters:
```
Comment (nbirillo) [inline scripts/src/utils/models/script_parameters.py:8]: Should classes from this file be marked as frozen?

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great, thank you! I have only one small comment

File: scripts/src/utils/models/script_parameters.py (hunk 1 above)
Comment (eartser) [reply scripts/src/utils/models/script_parameters.py:9]: Sure, that makes sense, thanks!

**Review by nbirillo — APPROVED** [review]
Great!

**Pull request merged**